pgcli/config.py

```diff
@@ -9,7 +9,7 @@
 
 def load_config(filename, default_filename=None):
     filename = expanduser(filename)
-    parser = ConfigParser()
+    parser = ConfigParser(interpolation=None)
```

Comment:
Let's replace this with `RawConfigParser()` because ConfigParser in Python 2 does not take a keyword arg `interpolation`.